bodhi/client/__init__.py

```diff
@@ -272,6 +272,28 @@ def composes():
     pass  # pragma: no cover
 
 
+@composes.command(name='info')
+@handle_errors
+@click.option('--release', required=True, help='The name of the release that is being composed.')
+@click.option('--request', default='testing',
+              help='The request of the release that is being composed.',
+              type=click.Choice(['testing', 'batched', 'obsolete', 'unpush', 'revoke', 'stable']))
```

Comment:
There's only two options for a compose in practice: ```testing``` and ```stable```.

```suggestion
              type=click.Choice(['testing', 'stable']))
```